scripts/ci-plutus-benchmark.sh

```diff
@@ -36,6 +39,13 @@ else
    git checkout "$PR_BRANCH"
 fi
 
+if [ -z "$CAPABILITY_NUM" ] ; then
```

Comment:
Is there any way of setting this from GitHub?

Reply:
Easiest way would be to:
`/benchmark BENCH_NAME CAP_NUM` 
Instead of
`/benchmark BENCH_NAME`
Sounds good?